test/ecs/crdt-rules-gset.spec.ts

```diff
@@ -258,11 +258,6 @@ describe('Conflict resolution rules for GrowOnlyValueSet based components', () =
     component.entityDeleted(0 as Entity, false)
     expect(component.get(0 as Entity)).toEqual(new Set([]))
   })
-
-  it('DELETE an existent entity clears out its value', () => {
-    component.entityDeleted(0 as Entity, false)
-    expect(component.get(0 as Entity)).toEqual(new Set([]))
-  })
```

Comment:
wat?

Reply:
it's a duplicated test

https://github.com/decentraland/js-sdk-toolchain/blob/28cb54f0e017f80c02592ebe17b35faaef720882/test/ecs/crdt-rules-gset.spec.ts#L257-L265